packages/flare/python/cron_job_ingest_events.py

```diff
@@ -1,23 +1,27 @@
 import json
-import sys
 import os
+import sys
+
+from datetime import date
+from typing import Any
 from typing import Optional
 
+
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), "vendor"))
 import vendor.splunklib.client as client
 
 from flare import FlareAPI
 
+
 APP_NAME = "flare"
 HOST = "localhost"
 SPLUNK_PORT = 8089
 REALM = APP_NAME + "_realm"
 KV_COLLECTION_NAME = "feednext"
```

Comment:
May I ask what this name is? did we select this name?

Reply:
We did during the PoC, will change it to something more obvious